From: Your Name <2901318208@qq.com>
Date: Thu, 12 Sep 2025 00:00:00 +0000
Subject: [PATCH 1/2] security: kconfig: Add source for Baseband Guard

Integrate the Kconfig for the Baseband Guard LSM into the main
security menu.

Signed-off-by: Your Name <your.email@example.com>
---
 security/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/security/Kconfig b/security/Kconfig
index 1234567..abcdefg 100644
--- a/security/Kconfig
+++ b/security/Kconfig
@@ -21,6 +21,8 @@ config SECURITYFS
 	
 source "security/selinux/Kconfig"
 source "security/smack/Kconfig"
+source "drivers/baseband_guard/Kconfig"
 
 config TOMOYO
 	bool "TOMOYO Linux"
